licensedb/licensedb.go

```diff
@@ -53,3 +53,8 @@ func Detect(fs filer.Filer) (map[string]api.Match, error) {
 	}
 	return licenses, nil
 }
+
+// Preload database with licenses
```

Comment:
As it extends the API surface, IMO it would be nice to add a few words here to clarify the contract - that it is an optimization and the client does need to call this first, as the rest of the APIs are doing that lazily anyway, or some words to that effect.

WDYT?

Reply:
Sure, will fix that